README.md

```diff
@@ -523,21 +523,28 @@ can install some projects that depend on MATLAB, in particular:
  * The [WB-Toolbox](https://github.com/robotology/WB-Toolbox) Simulink toolbox,
  * The [whole-body-controllers](https://github.com/robotology/whole-body-controllers) Simulink-based balancing controllers.
 
-
 To use this software, you can simply enable its compilation using the `ROBOTOLOGY_USES_MATLAB` CMake option.
 Once this software has been compiled by the superbuild, you just need to add some directories of the robotology-superbuild install (typically `$ROBOTOLOGY_SUPERBUILD_ROOT/build/install`) to [the MATLAB path](https://www.mathworks.com/help/matlab/matlab_env/what-is-the-matlab-search-path.html).
 In particular you need to add to the MATLAB path the `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/mex` directory and all the subdirectories `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/WBToolbox`.
 
-As an example, you could add this line to your MATLAB script that uses the robotology-superbuild matlab software:
+#### Start MATLAB from the launcher or the application menu
```

Comment:
I think this may be misleading. If MATLAB is launched from launcher of the application menu, who is setting the `ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX` env var? @gabrielenava @nunoguedelha any opinion?

Reply:
I personally never used that script, I know that @gabrielenava does use it, let's wait his feedback.

Reply:
Btw, those instructions were there before, probably they had to be updated.

Reply:
Note that as far as I know [this issue](https://github.com/robotology/codyco-superbuild/issues/141#issuecomment-395134655) still occurson Ubuntu 18.04, therefore even running the `setup.sh` file the user will have to manually address this before being able to use the toolbox properly. 

Also, there are two things that should be properly documented:

1) again: that if the user choose to run the `startup.m` or `startup_robotology_superbuild.m` it is required to always run Matlab from the folder where the `pathdef.m` is;

2) I recently decided to convert the [whole-body-controllers library](https://github.com/robotology/whole-body-controllers/tree/master/library/matlab-wbc/%2Bwbc) in a package that can be pointed by the matlab path and used by any Simulink model in any folder. This implies that the user that wants to use `whole-body-controllers` also need to follow [these instructions](https://github.com/robotology/whole-body-controllers#installation-and-usage). Note that there are 2 possibilities: manually add the library to the matlabpath or run the yet-another-one [startup.m](https://github.com/robotology/whole-body-controllers/blob/master/config/startup.m) script. If the user does the following **combo**:

- **WBToolbox**: set the required Matlab path using the setup.sh or manually add the required folders to the MATLABPATH;

- **wbc library**: run the `startup.m`;

then **if** the user **does not run** matlab from the folder where the `pathdf.m` is, he will be able to use `WBToolbox` but not `whole-body-controllers`. 


I think from the Simulink controllers point that's it.

Reply:
> Note that as far as I know this issue still occurs on Ubuntu 18.04, therefore even running the setup.sh file the user will have to manually address this before being able to use the toolbox properly.

Can we please open new issue if problem exist instead of posting in closed issues?  :''( 

> I recently decided to convert the whole-body-controllers library in a package that can be pointed by the matlab path and used by any Simulink model in any folder. This implies that the user that wants to use whole-body-controllers also need to follow these instructions. 

I do not get why this is necessary, but this should be moved to a new  issue.

Reply:
> Can we please open new issue if problem exist instead of posting in closed issues? :''(

Sorry, I was confused by the old issue, the problem is discussed in https://github.com/robotology/robotology-superbuild/issues/64 .

Reply:
I just have a question, if two projects need the `pathdef.m`, is there any possibility that the second project overrides the file created by the first?